blaze/compute/core.py

```diff
@@ -367,8 +371,14 @@ def swap_resources_into_scope(expr, scope):
 
 
 @dispatch(Expr, dict)
-def compute(expr, d, **kwargs):
-    """ Compute expression against data sources
+def compute(expr, d, return_type='native', **kwargs):
+    """ Compute expression against data sources.
+
+    Args:
```

Comment:
Could you edit the docstring to fit with the other docstring styles?  Like [join, for example ](https://github.com/blaze/blaze/blob/master/blaze/expr/collections.py#L409-L446).